src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
+            result = a + b;
+        end
+        
+        function result = subtract(a, b)
+            result = a - b;
+        end
+        
+        function result = multiply(a, b)
+            result = a * b;
```

Comment:
The divide method should handle cases where 'b' is not a number, in addition to checking for zero.